src/components/pages/pseudochoice.tsx

```diff
@@ -4,6 +4,46 @@ import React from 'react';
 //   onCorrect: () => void;
 // }
 
+function MoveTurnComponent(input){
+  return(
+    <><div>
+      <p>Move forward {input.number} times</p>
+    </div>
+    <div>
+      <p>Turn {input.direction}</p>
+    </div></>
+  );
+}
+
+function AllChoices(input){
+  return(
+    <div>
+      <button>
+        <MoveTurnComponent
```

Comment:
If we pass in as 2 arrays or a 2D array, we can use `map` to display each component. This is more efficient and also doesn't limit the number of instructions.